### PR TITLE
Implemented the bottom sheet, and cleaned the code in MainActivity.kt file. Merged multiple commits.

### PR DESCRIPTION
**Pull request by Adithya2907**
### Description
Resolved merge conflicts. Merged multiple commits.

Please ensure the following and put a check accordingly

- [ ] Ran checks with `./gradlew ktlintCheck` command

- [x] Squashed multiple commits (_if irrelevant_) into one commit

### CODE REVIEW

**Comment by antailbaxt3r** [comment]
You have used binding and findViewById both. Use neither. Instead use the kotlinx viewbinding

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -21,7 +20,7 @@ xmlns:app="http://schemas.android.com/apk/res-auto">
             android:layout_width="match_parent"
             android:layout_height="match_parent">
 
-            <include layout="@layout/bottom_sheet" />
+            <include layout="@layout/bottom_sheet"/>
```
Comment (antailbaxt3r) [inline app/src/main/res/layout/activity_main.xml:23]: Unnecessary change

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -36,18 +35,19 @@ xmlns:app="http://schemas.android.com/apk/res-auto">
         android:id="@+id/main_frame_layout"
         android:layout_width="match_parent"
         android:layout_height="match_parent">
+
```
Comment (antailbaxt3r) [inline app/src/main/res/layout/activity_main.xml:38]: Unnecessary change

**Review by antailbaxt3r — CHANGES_REQUESTED** [review]

**Comment by antailbaxt3r** [comment]
Also, squash commits

**Comment by Adithya2907** [comment]
Can you please do the squashing alone?? Coz whenever I do it, it reverts back to the old version, and says "this branch is 5 commits behind." This happens every time for me

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (Adithya2907) [reply app/src/main/res/layout/activity_main.xml:23]: These changes occur automatically, because I use Android Studio's build-in formatter, using the Ctr+Alt+L key. And that formats the document this way, with all the spaces and extra lines.
